test/sequential/test-deprecation-flags.js

```diff
@@ -39,10 +45,22 @@ execFile(node, traceDep, function(er, stdout, stderr) {
   console.log('trace ok');
 });
 
-execFile(node, [depUserland], function(er, stdout, stderr) {
+execFile(node, [depUserlandFunction], function(er, stdout, stderr) {
   console.error('normal: testing deprecated userland function');
   assert.equal(er, null);
   assert.equal(stdout, '');
   assert(/deprecatedFunction is deprecated/.test(stderr));
```

Comment:
FWIW, an easier way to test for the deprecation warning would be to listen for `warning` events...

```js
process.on('warning', (warning) => {
  if (warning.name === 'DeprecationWarning') {
    // ...
  }
});
```

But that doesn't need to be fixed here at all.. just pointing it out